docs/versionhistory.rst

```diff
@@ -4,6 +4,11 @@ Version history
 This library adheres to
 `Semantic Versioning 2.0 <https://semver.org/#semantic-versioning-200>`_.
 
+**UNRELEASED**
+
+- Avoid creating reference cycles when type checking unions
+  (`#408 <https://github.com/agronholm/typeguard/pull/408>`_)
```

Comment:
If there was no issue being resolved by this, then the link should go away.

Reply:
Sorry about that, removed!